tests/ext/test_updater.py

```diff
@@ -692,7 +693,7 @@ async def delete_webhook(*args, **kwargs):
     @pytest.mark.parametrize("ext_bot", [True, False])
     @pytest.mark.parametrize("drop_pending_updates", [True, False])
     @pytest.mark.parametrize("secret_token", ["SecretToken", None])
-    @pytest.mark.parametrize("unix", [None, True])
+    @pytest.mark.parametrize("unix", [None, 1, 2])
```

Comment:
```suggestion
    @pytest.mark.parametrize("unix", [None, "file_path", "socket_object"])
```
can we make this more descriptive?